src/components/ButtonComponents/SpecialButtons/Specials.js

```diff
@@ -1,17 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
 
 //import any components needed
+import SpecialButton from './SpecialButton';
 
 //Import your array data to from the provided data file
+import { specials } from '../../../data';
 
 const Specials = () => {
   // STEP 2 - add the imported data to state
+  const [specialsState, setSpecialsState] = useState(specials);
```

Comment:
for setting the state values, you don't need the second value in the array if we're not looking to update the data